contrib/README.md

```diff
@@ -0,0 +1,24 @@
+contrib
+=======
+
+## Overview
+
+This consists of extra optional tools which may be useful when working with
+dcrd and related software.
+
+## Contents
+
+### Example Service Configurations
+
+- [rc.d](services/rc.d/dcrd)  
+  Provides an example `rc.d` script for configuring dcrd as a background service
+  on operating systems that use the rc.d system for service management.
+
+- [Service Management Facility](services/smf/dcrd.xml)  
+  Provides an example XML file for configuring dcrd as a background service on
+  operating systems that use SMF for service management.  This is typically
+  Solaris/OpenSolaris.
```

Comment:
OpenSolaris isn't around anymore, illumos is the name of the fork.

Reply:
Updated to say:

```
  Provides an example XML file for configuring dcrd as a background service on
  illumos.  It also serves as a good starting point for other operating systems
  that use use SMF for service management.
```